kuma/users/tasks.py

```diff
@@ -58,3 +61,23 @@ def send_welcome_email(user_pk, locale):
             )
             email.attach_alternative(content_html, "text/html")
             email.send()
+
+
+@task
+@skip_in_maintenance_mode
+def send_payment_received_email(stripe_customer_id, locale, timestamp, invoice_pdf):
+    user = get_user_model().objects.get(stripe_customer_id=stripe_customer_id)
+    locale = locale or settings.WIKI_DEFAULT_LANGUAGE
+    context = {
+        "payment_date": datetime.datetime.fromtimestamp(timestamp),
+        "manage_subscription_url": absolutify(reverse("recurring_payment_management")),
+        "faq_url": absolutify(reverse("recurring_payment_subscription")),
+        "contact_email": settings.CONTRIBUTION_SUPPORT_EMAIL,
+        "invoice_pdf": invoice_pdf,
+    }
+    with translation.override(locale):
+        subject = render_email("users/email/payment_received/subject.ltxt", context)
+        # Email subject *must not* contain newlines
+        subject = "".join(subject.splitlines())
```

Comment:
Why not just `subject = subject.strip()` which would do the same thing. No?

Reply:
I don't think it would. It would not take care of newlines between non-space characters, which I think this was addressing (I followed the other email tasks example here).